src/components/SelectButton.tsx

```diff
@@ -0,0 +1,22 @@
+const SelectButton = () => {
+  return (
+    <div className="flex items-center justify-center">
+      <h2 className="text-bold ml-4">Branches:</h2>
+      <div className="dropdown m-4 w-3/4">
+        <div
+          tabIndex={0}
+          role="button"
+          className="btn w-full border-white text-left"
+        >
+          Show All
+        </div>
+        <ul
+          tabIndex={0}
+          className="menu dropdown-content z-[1] w-full rounded-box bg-base-100 p-2 shadow"
```

Comment:
Is there a reason why you are using an arbitrary value for the z-index ` z-[1] ` rather than the Tailwind's z-index utility classes ` z-10, z-20 ... ` ? 
If it's because you are using specific z-indices elsewhere then it's all good 😊

Reply:
This was a DaisyUI thing. Don't know why they added it. Let me see if it still works without it

Reply:
If that's the case then I guess just leave it